code/swb_editor/OffsetEditor.razor

```diff
@@ -29,7 +29,7 @@
 		<SliderEntry min="-360" max="360" step="0.01" name="Roll" Value:bind=@Roll></SliderEntry>
 
         <text class="subTitle">FOV</text>
-		<SliderEntry min="1" max="179" step="1" name="FOV" Value:bind=@FOV></SliderEntry>
+		<SliderEntry min="-160" max="160" step="1" name="FOV" Value:bind=@FOV></SliderEntry>
```

Comment:
Why -160? the range is 0 to 179 before the weapon disappears

Reply:
What was the reasoning to changing it to a multiplier system? Because blender and other modeling tools use fixed values for FOV which might make it confusing when importing models to use.